### ISSUE TITLE
AVT 3 - React Tabs Component, VO on macOS and iOS has issues with reading order

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3467]
## Environment
macOS 10.14.5
Chrome 75.0.3770.100
VoiceOver

Note: Same issue with iOS, except navigation is using gestures instead of the keyboard. 
iOS 12.3.1
Safari
VoiceOver

## Description
1. Tab to the Tabs Component and focus is on Tab label 2 and it is announced as expected.
2. Use Arrow keys to navigate to Tab label 1 it is announced as expected.
3. Use Control - Option - Arrow to read content. VO announces, Tab label 2, Tab label 3, Tab label 4, Content for first tab goes here.
**Expected Result:** Tab content should receive focus after the Tab Label 1 is selected, and Content for first tab goes here. should be announced by VO.
Fails: [WCAG 2.0 1.3.2 Meaningful Sequence](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

**Comment by snidersd** [comment]
@dakahn I just retested this issue when testing #2487 and it is working as expected.
